scripts/waifulib/xcompile.py

```diff
@@ -240,9 +241,22 @@ def strip(self):
 			return os.path.join(self.gen_binutils_path(), 'llvm-strip')
 		return os.path.join(self.gen_binutils_path(), 'strip')
 
+	def objcopy(self):
+		if self.is_host() and not self.ndk_binutils:
+			environ = getattr(self.ctx, 'environ', os.environ)
```

Comment:
You don't need `getattr` here. 

Context already should have environ (we're literally overriding compilers and tools through it)